sanic/cli/app.py

```diff
@@ -76,7 +85,7 @@ def run(self, parse_args=None):
         elif parse_args == ["-v"]:
             parse_args = ["--version"]
 
-        if not legacy_version:
+        if not legacy_version and not self.inspecting:
```

Comment:
This addition shouldn't be needed. We `return` above `if self.inspecting`

Reply:
Yup. Legacy. TY.